### ISSUE TITLE
Opt-in/-out for tagging in photos

### ISSUE THREAD

**Issue by rinkp** [GEWIS/gewisweb#1410]
A member suggested that it might be desirable for users to be able to opt-out to being tagged in photos, for example if they have a job which requires them to do so. In principle, all tags are now kept up to 2 years after a subject is no longer a Member of Graduate, but one may want to remain a Graduate without being tagged.

**Comment by tomudding** [comment]
Good idea. Maybe we can actually revive part of #148 when we implement this (e.g., don't show birthday on frontpage). Though I cannot give any guarantees on when this can be implemented.
